services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/util/DemoUtil.java

```diff
@@ -0,0 +1,202 @@
+package com.futurewei.alcor.netwconfigmanager.util;
+
+import com.futurewei.alcor.netwconfigmanager.entity.HostGoalState;
+import com.futurewei.alcor.schema.*;
+
+import java.util.Map;
+
+public class DemoUtil {
+
+    final public static String aca_node_one_ip = "10.213.43.92";
+    final public static String aca_node_two_ip = "10.213.43.93";
+    final public static String vpc_id_1 = "2b08a5bc-b718-11ea-b3de-111111111112";
+    final public static String port_id_1 = "11111111-b718-11ea-b3de-111111111112";
+    final public static String port_id_2 = "13333333-b718-11ea-b3de-111111111114";
+    final public static String subnet_id_1 = "27330ae4-b718-11ea-b3df-111111111113";
+
+    public static void populateHostGoalState(Map<String, HostGoalState> hostGoalStates){
+
```

Comment:
@xieus Should we put this class in the test folder?

Reply:
Good point. Let me do it now.

Reply:
Unfortunately, this code is used in the java folder for demo purpose. It is not ready to move to test folder as of now otherwise the production code (under java) won't be able to access it. Will move in next PR then.